### DISCUSSION TITLE
sessions aren't being synced across browsers, and can't import sessions either

### DISCUSSION THREAD

**Discussion by EthasTheSkunk** [sienori/Tab-Session-Manager#1344] [Q&A]
hey, so, I'm trying export my sessions from Edge over to Firefox.

I tried exporting all of the sessions, and I have the file, but when I try importing it, nothing happens.

I can export individual sessions, in which case it'll open and lets me import, but that's only a work around.

also, trying to sync, every time I do so, it doesn't sync up with other each other.
is the cloud sync only for per browser?

also how do I import all of the sessions at once?

update: I'm literally seeing the extension literally remove a session I've imported and saw saved. I literally double checked, and while it was syncing, it literally just removed it despite me importing it. it's doing it to 1 session, but not the other.

**Comment by awprawpr** [comment]
Does the lack of replies imply the developer has deserted the app?
